modules/nextflow/src/main/groovy/nextflow/k8s/K8sTaskHandler.groovy

```diff
@@ -379,28 +381,6 @@ class K8sTaskHandler extends TaskHandler {
         return false
     }
 
-    protected void savePodLogOnError(TaskRun task) {
-        if( task.isSuccess() )
-            return
-
-        if( errorFile && !errorFile.empty() )
-            return
-
-        final session = executor.getSession()
-        if( session.isAborted() || session.isCancelled() || session.isTerminated() )
-            return
-
-        try {
-            final stream = useJobResource()
-                    ? client.jobLog(podName)
-                    : client.podLog(podName)
-            Files.copy(stream, task.workDir.resolve(TaskRun.CMD_LOG))
-        }
-        catch( Exception e ) {
-            log.warn "Failed to copy log for ${resourceType.lower()} $podName", e
-        }
-    }
-
     protected int readExitFile() {
```

Comment:
Why this was removed?

Reply:
This method would save the command log for failed pods, so it's no longer needed.